src/test/java/ru/practicum/shareit/jpa/CommentDataJpaTest.java

```diff
@@ -0,0 +1,61 @@
+package ru.practicum.shareit.jpa;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+import ru.practicum.shareit.booking.Booking;
+import ru.practicum.shareit.booking.BookingRepository;
+import ru.practicum.shareit.booking.Status;
+import ru.practicum.shareit.item.CommentRepository;
+import ru.practicum.shareit.item.ItemRepository;
+import ru.practicum.shareit.item.model.Comment;
+import ru.practicum.shareit.item.model.Item;
+import ru.practicum.shareit.user.UserRepository;
+import ru.practicum.shareit.user.model.User;
+
+import java.time.LocalDateTime;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DataJpaTest
+public class CommentDataJpaTest {
+
+    @Autowired
+    private TestEntityManager entityManager;
+    @Autowired
+    private CommentRepository commentRepository;
+    @Autowired
+    private UserRepository userRepository;
+    @Autowired
+    private ItemRepository itemRepository;
+
+    @Autowired
+    private BookingRepository bookingRepository;
+
+    @Test
+    void testItemRequestDto() throws Exception {
+        LocalDateTime time = LocalDateTime.now().withNano(000000);
```

Comment:
Будет работать и с простым 0
```suggestion
        LocalDateTime time = LocalDateTime.now().withNano(0);
```